src/classes/HH_Container_LCTRL.cls

```diff
@@ -37,6 +37,28 @@
 public class HH_Container_LCTRL {
     
     /*******************************************************************************************************
+    * @description trick the packaging spider to pickup references to custom labels only used by our
+    * lightning components.
+    */
+    public static void packageLabels() {
```

Comment:
I'm looking at the diff that appears when I click the 'Files Changed' tab at the top of this pull request. What the heck??  I wonder how much of what I reviewed is out of date, then...

Why is the diff on the PR so messed up?